platform/android/src/offline/offline_manager.cpp

```diff
@@ -211,6 +215,7 @@ void OfflineManager::registerNative(jni::JNIEnv& env) {
         METHOD(&OfflineManager::invalidateAmbientCache, "nativeInvalidateAmbientCache"),
         METHOD(&OfflineManager::clearAmbientCache, "nativeClearAmbientCache"),
         METHOD(&OfflineManager::setMaximumAmbientCacheSize, "nativeSetMaximumAmbientCacheSize"),
+        METHOD(&OfflineManager::setAutopack, "setAutopack"),
```

Comment:
It seems based on the current naming convention, the name of this API should start with `native*`

Otherwise, why do some of these APIs start with `native*` and not others?

Reply:
`native*` are the methods that are not exposed directly, i.e. require JAVA wrappers. `setAutopack` (`runPackDatabaseAutomatically`) is simple enough to be exposed as it is.